src/test/java/uc/dei/mse/supportivecare/domain/OutcomeTest.java

```diff
@@ -19,5 +19,8 @@ void equalsVerifier() throws Exception {
         assertThat(outcome1).isNotEqualTo(outcome2);
         outcome1.setId(null);
         assertThat(outcome1).isNotEqualTo(outcome2);
+        outcome1.setLink("");
```

Comment:
This change is not needed since the test valid valid 2 objects are equals or not, and they are identified by the Id

Reply:
do you mean to delete line 22?

Reply:
Delete those changes (the 3 lines)